lib/mix/tasks/ruby.ex

```diff
@@ -0,0 +1,108 @@
+defmodule Mix.Tasks.Thrifter.Ruby do
+  use Mix.Task
+
+  @gen_path "gen/ruby-client"
+  @client_name "test_client"
+
+  @lib_path "#{@gen_path}/lib"
+  @bin_path "#{@gen_path}/bin"
+  @templates_path "templates/ruby"
+
+  @version "0.1.0"
```

Comment:
I changed that in https://github.com/renderedtext/thrifter/pull/9.
Please check if something else is missing.